### PR TITLE
Hiring Section

### PR DESCRIPTION
**Pull request by bogren**
I feel a bit stuck and would like feedback on the text, what's missing, and what parts we should remove.

### CODE REVIEW

File: Hiring/Hiring-framework.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+## Hiring Manager
+
+### Reference check
```
[the remark this answers is not in the document]
Comment (bogren) [reply Hiring/Hiring-framework.md:3]: No I had it included first but if P&C do the reference checks we can leave it out.

**Review by Svedenmark — COMMENTED** [review]
Great write-up

File: Hiring/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+## Hiring pages
+
+Collection of Hiring related pages.
+
+* [Interviewing](Interviewing.md)
+* [Teamtailor](Teamtailor.md)
+* [Interview Kit](Interview-kit.md)
```
Comment (lhansford) [inline Hiring/README.md:7]: This link needs updating I guess

**Review by lhansford — APPROVED** [review]
Very nice work Emil!

File: Hiring/Interviewing.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+* Set expectations on how the interview will run.
+
+* Cover the prepared topics/questions without being mechanical.
+
+* Setup the interview plan in Teamtailor ahead of the interview to ensure all the areas you want to cover are in the scorecard.
+
+* Try to cover the prepared topics, but keep the conversation smooth - it is ok to skip around.
+
+* Redirect the conversation as needed - you are in charge of keeping the conversation focused and timely.
+
```
Comment (mattias-lundell) [inline Hiring/Interviewing.md:50]: I think it is important to spare a couple of minutes at the end to wrap up and give the candidate some space for asking questions etc.

**Review by mattias-lundell — APPROVED** [review]

**Pull request merged**